### ISSUE TITLE
Notifications are displayed on a monitor other than the main application window

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#35790]
### Description

If user has more than one monitor, DBeaver shows the notification on the first(?) monitor in the right bottom corner of the screen, when the DBeaver's main window is located on the second monitor.
It is more convenient when notification is displayed on the same monitor where the main window of the application is located or close enough to main window.
Otherwise the notification could have been easily missed (it disappears automatically by design and can be not noticed in cases when monitors' width is large).

Please also consider that the main window can be stretched across more than one monitor.
In this case, perhaps, it is OK to show the notification in the right bottom corner of the monitor where the upper-left point of the main application window is located.

**How to trigger the notification for debug purpose?**
- I set the property of the connection: "Close idle connection after (seconds)" to 1 second, open new tab in this connection and wait for the notification.

![img](https://github.com/user-attachments/assets/8e38afc1-cef6-44ea-95ad-2618e1f4b0f3)

### DBeaver Version

Enterprise Edition 24.2.2-202409192322

### Operating System

Windows 10

**Comment by E1izabeth** [comment]
Yes, I experienced it too. Thank you for reporting this behavior.
